lint-staged.config.mjs

```
@@ -0,0 +1,7 @@
+export default {
+  "*.{ts,tsx,vue,js,jsx}": [
+    "eslint --fix",
+    () => "pnpm typecheck:fixed",
+    () => "pnpm typecheck:baseline"
+  ]
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)